.github/CODEOWNERS

```diff
@@ -73,3 +73,7 @@
 # framework changes (excluded by not setting an owner next to it- no owner)
 /performance @nathaniel-may
 /performance/projects
+
+### ARTIFACTS
+
+/schemas/dbt                    @dbt-labs/cloud-artifacts
```

Comment:
https://github.com/orgs/dbt-labs/teams/cloud-artifacts

Reply:
From the [GH docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax): 
> Users and teams must have explicit write access to the repository, even if the team's members already have access.

@leahwicz - are we okay granting write access to @dbt-labs/cloud-artifacts, assuming they don't have it already?

Reply:
Yeah that is fine. We have PR reviews enforced so its not like an accidental commit can be pushed